style.css

```diff
@@ -397,7 +424,7 @@ button a {
 
 button {
   color: #396df2;
-  font-family: "Poppins", sans-serif;
+  font-family: 'Poppins', sans-serif;
   font-style: normal;
   font-weight: 500;
   font-size: 17px;
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
As you can see from the difference below, In the Figma design the `about myself title`, `the paragraph`, `let's connect` and `get my resume` button lies on the same left line. there is no margin left for the `get my resume` button.

| Your Design |Figma Design |
| :---:   | :---: | 
| ![image](https://user-images.githubusercontent.com/92303891/205455032-d746d37b-1c2d-4018-a2f1-0ec3fd3151fa.png)| ![image](https://user-images.githubusercontent.com/92303891/205455060-5683c142-674c-4eca-a4e4-e5e243fce8d8.png)|


- [ OPTIONAL ] Although the previous reviewer did not specify this, it would be better if you make the background color of the about section looks the same as the  Figma design.